modules/test-ui/src/utils/crypto.ts

```diff
@@ -0,0 +1,32 @@
+import { HexString, PublicKey, PrivateKey } from "@connext/vector-types";
```

Comment:
Let's move this stuff into the `@connext/vector-utils` package instead of in the test-ui.

Reply:
Ahh didn't notice we already had utils. Done